src/tutorialGBIF_occurrence_maps.Rmd

```diff
@@ -0,0 +1,175 @@
+                                  max(spec$decimalLongitude, na.rm = TRUE))))
+longitude_min <- min(unlist(map(occurrences, 
+                                function(spec) 
+                                  min(spec$decimalLongitude, na.rm = TRUE))))
+
+latitude_max <- max(unlist(map(occurrences, 
+                                function(spec) 
+                                  max(spec$decimalLatitude, na.rm = TRUE))))
+latitude_min <- min(unlist(map(occurrences, 
+                                function(spec) 
+                                  min(spec$decimalLatitude, na.rm = TRUE))))
+purrr::map(names(occurrences), function(spec) 
+  qmplot(x = decimalLongitude,
+         y = decimalLatitude, 
+         data = occurrences[[spec]][1:1000,], 
+         maptype = "toner-lite", 
+         xlim = c(longitude_min, longitude_max), 
+         ylim = c(latitude_min,latitude_max),
+         main = spec,
+         color = I("blue")))
+```
+
+### Monthly occurrence distribution
+For some species can be interesting to see the monthly occurrence distribution:
+```{r}
+months <- c(1:12)
+occurrences_per_month <- map(months, function(m) map(occurrences, function(x) x %>% filter(month == m)))
+counts_per_months <- map(species, 
+                         function(i) unlist(
+                           map(occurrences_per_month, 
+                               function(x) 
+                                 nrow(x[[i]]))))
+counts_per_months <- set_names(counts_per_months, nm = species)
+counts_per_months_df <- tidyr::gather(data.frame(counts_per_months), species)
+counts_per_months_df$month <- rep.int(1:12, length(species))
+ ggplot(counts_per_months_df, aes(fill = species, y = value, x = month)) +
+   geom_bar(position="dodge", stat="identity") +
+   scale_x_continuous(breaks=seq(1,12,1), 
+                     labels=c("Jan", "Feb", "Mar", "Apr", "May", "Jun",
+                              "Jul", "Aug", "Sept", "Oct", "Nov",
+                              "Dec"))
+```
+
+### Occurrence geographical distribution based on UTM-10 squares
+Import Belgium UTM-10 square polygon data:
+```{r}
+eu_10grid <-  readOGR("../data/input/EUgrid10.geojson")
+wgs_84 <- CRS("+init=epsg:4326") # CRS used by GBIF
+eu_10grid <- spTransform(eu_10grid, wgs_84)
+```
+
+Transform occurrence positions to spatial points:
+```{r}
+coord_occ <- purrr::map(occurrences, 
+                 function(spec) data.frame(cbind(spec$decimalLongitude,
+                                                 spec$decimalLatitude)))
+coord_occ <- purrr::map(coord_occ, function(spec) set_names(spec, c("lon", "lat")))
+occ_points <- map2(coord_occ, occurrences, 
+                   function(x, y) SpatialPointsDataFrame(
+                     coords = x, 
+                     data = y, 
+                     proj4string = wgs_84))
+```
+
+The function `over` is very useful for spatial JOINs. In our case it can be used to get the occurrence points contained in the square:
```

Comment:
Rewrite this in a more 'active' form. 

Start from the question you want to solve: *I'd like to know the number of occurrences for each UTM 10 gridcell*

Next, provide the steps to solve this:
1. Add the corresponding UTM gridcell to the data -> spatial join -> `over`
2. count per gridcell (hmm, this is actually the same as the counting for each month!)